lib/typings.d.ts

```diff
@@ -0,0 +1,10 @@
+export interface RetrievalTask {
+  cid: String;
+  providerAddress: String;
+  protocol: String;
+}
+
+export interface RoundDetails {
+  roundId: string; // BigInt
```

Comment:
can we use BigInts instead?

Reply:
This interface describes the data we are getting from the IPFS. JSON does not support BigInt, therefore we are transmitting this value as a string.